.github/workflows/lint-pr.yml

```diff
@@ -13,13 +13,38 @@ on:
       - synchronize
 
 permissions:
-  pull-requests: read
+  pull-requests: write
```

Comment:
The JS SDK apparently manages to get by without this permission. Can we confirm it's required?

See https://github.com/open-feature/js-sdk/blob/main/.github/workflows/lint-pr.yml

Reply:
👀 I will look into it

Reply:
from the docs (https://github.com/marocchino/sticky-pull-request-comment#basic) it is needed.
Looking into GH Action docs (see https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests), adding a comment requires this permission.
I'll create an issue for the JS-SDK :)

Reply:
created https://github.com/open-feature/js-sdk/issues/827